src/containers/Delete/index.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import Report from "../../components/Report";
+import {connect} from 'react-redux';
+import {Button} from 'react-materialize';
+import {bindActionCreators} from 'redux';
+import {ActionCreators} from '@actions';
+
+class Delete extends Component {
+
+    handleButtonClick() {
+        this.props.deleteActiveReport();
+    }
+
+    render() {
+        return (
+            <div className="prettier">
+                <h5>Selected report:</h5>
+                <hr/>
+
+                {!this.props.reports.activeReport ?
```

Comment:
Good practice for toggling state depending on current props value. I often also use the class method on ternary expression to render content conditionally, like so:

...
{this.props.reports.activeReport ? this.renderNothingToDisplayComponent() : this.renderReportsContent()}
...

Then we are able to take advantage of OOP and stateful React components.